apt-repository/Dockerfile.reprepro

```diff
@@ -0,0 +1,3 @@
+FROM debian:buster
+
+RUN apt-get -q update && apt-get install -q -y reprepro git
```

Comment:
Is this producing warnings? Do we want to set the `DEBIAN_FRONTEND="noninteractive"`?